api/envoy/config/filter/http/fault/v2/fault.proto

```diff
@@ -41,12 +48,14 @@ message FaultAbort {
     HeaderAbort header_abort = 4;
   }
 
+  GrpcError grpc_error = 5;
```

Comment:
this doesn't look right. Shouldn't the grpc_error be in the oneof above?

Reply:
Yep, it should. I kept it separate to support the scenario where user wants to set both http_status and grpc_status but this is not required, as http_status should always be 200 when grpc_status is set.